e2e2/test/cases/nvidia/manifests/job-unit-test-single-node.yaml

```diff
@@ -0,0 +1,28 @@
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: unit-test-job
+  labels:
+    app: unit-test-job
+spec:
+  template:
+    metadata:
+      labels:
+        app: unit-test-job
+    spec:
+      containers:
+      - name: unit-test-container
+        image: public.ecr.aws/o5d5x8n6/weicongw:nvidia
```

Comment:
How will we swap this out?

Reply:
This was for my dev test, and I've changed it to use template variables. The image comes from `e2e2/test/images/nvidia/Dockerfile`, and it's almost the same as our previous NCCL test image. 
I am also thinking of using the same docker image for all NVIDIA tests and putting the Dockerfile in `e2e2/test/images`. If you agree with this approach, I'll remove `e2e2/test/images/Dockerfile.aws-efa-nccl-tests` in the next PR.

Reply:
Yeah that makes sense to me, we should be able to get by with a single image. let's do that in a followup